src/runner/browser-job.ts

```diff
@@ -214,13 +214,20 @@ export default class BrowserJob extends AsyncEventEmitter {
         return this._testRunControllerQueue;
     }
 
+    private _getlastIndexFixture ( fixtureId: string ): number {
+        const lastIndexFixture = this._testRunControllerQueue.findIndex(el => el.test.fixture?.id !== fixtureId);
+
+        if (!~lastIndexFixture)
```

Comment:
The following statement is much easier to read:
```suggestion
        if (lastIndexFixture === -1)
```